### PR TITLE
Issue #303: Made uninstallation scripts documentation updates in 5.3.1 directory

### PR DESCRIPTION
**Pull request by tst-ccamp**
Implements #303

### CODE REVIEW

**Comment by tst-ccamp** [comment]
OK, here they are:
https://tst-ccamp.github.io/earthenterprise/geedocs/5.3.1/answer/7160008.html
https://tst-ccamp.github.io/earthenterprise/geedocs/5.3.1/answer/3499964.html
https://tst-ccamp.github.io/earthenterprise/geedocs/5.3.1/answer/4492617.html

File: docs/geedocs/5.3.1/answer/7160008.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h2>Release notes: Open GEE 5.3.1</h2>
   <h5>
     <p id="overview_5.3.1">New Features</p>
   </h5>
-    <p><strong>Placeholder for first new feature in 5.3.1. </strong>Put description here</p>
+    <p><strong>Uninstallation scripts are now copied over to the installation at <code>/opt/google/install</code>.  </strong>This will allow the user to uninstall from the installation instead of having to have a copy of the source code on hand.</p>
   <h5>
     <p id="supported_5.3.1">Supported Platforms</p>
   </h5>
```
Comment (tst-nfarah) [inline docs/geedocs/5.3.1/answer/7160008.html]: minor: copyright year has 2018 in it. It should be 2019 only.

File: docs/geedocs/5.3.1/answer/3499964.html [hunk 2]
```diff
@@ -23,8 +23,7 @@ <h2>Uninstalling</h2>
 <li>Stop the Google Earth Enterprise Server, if you have a previous version installed and it is running:
 <p><code>/etc/init.d/geserver stop</code></p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the script <code>uninstall_server.sh</code>.</li></ol></div>
-</ul><div class="footer"><p class="BackToTop"><a href="#top_of_file">Back to top</a> <hr /></p> <p class="copyright">&copy;2015 Google</p></div>
+<li>Navigate to the location at which you initially cloned the GEE git repo or to the installation location (<code>/opt/google</code>).</li>
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.1/answer/3499964.html]: This doesn't quite make sense to me:
image
Steps 4 and 5 are redundant. I think they should be combined.

File: docs/geedocs/5.3.1/answer/4492617.html [hunk 3]
```diff
@@ -31,14 +31,13 @@ <h2>Uninstalling</h2>
 <code>/etc/init.d/geserver stop</code>
 </p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the <code>uninstall_fusion.sh</code> and <code>uninstall_server.sh</code> scripts.</li></ol></div>
+<li>Navigate to the location at which you initially cloned the GEE git repo or to the installation location (<code>/opt/google</code>).</li>
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.1/answer/4492617.html]: Same comment as above.

**Comment by tst-ccamp** [comment]
@tst-lsavoie I decided to just take out the optional part about uninstalling from the git repo. As long as users have on method to uninstall, that's good enough. I think it is a lot easier to understand now.

File: docs/geedocs/5.3.1/answer/4492617.html [hunk 4]
```diff
@@ -31,14 +31,13 @@ <h2>Uninstalling</h2>
 <code>/etc/init.d/geserver stop</code>
 </p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the <code>uninstall_fusion.sh</code> and <code>uninstall_server.sh</code> scripts.</li></ol></div>
+<li>Navigate to the installation location (<code>/opt/google/install</code>).</li>
+<li>Run the script <code>uninstall_server.sh</code>.</li></ol></div>
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.1/answer/4492617.html]: This one is for both Fusion and server, so the instructions should say to run both scripts.

**Pull request merged**